### DISCUSSION TITLE
building qbit4.1.9.1 with lib 1.1.14

### DISCUSSION THREAD

**Discussion by Sparklingx** [userdocs/qbittorrent-nox-static#53] [Q&A] [answered]
How can I build qbit4.1.9.1 with lib 1.1.14? What should I change in your script? 
https://git.io/qbstatic

**Comment by userdocs** [comment]
https://userdocs.github.io/qbittorrent-nox-static/#/build-help

So something like:

```
~/qbittorrent-nox-static.sh all -qt release-4.1.9.1 -lt libtorrent-1_1_14
```

No promise it works. That's just how it's done.

**Reply by Sparklingx** [reply]
@userdocs 
Yap it shows an error.  Basically I could use your builds but I want to use custom patch. 

""

> checking whether the Boost::System library is available... yes
> checking for exit in -lboost_system... yes
> configure: Boost.System LIB: "-lboost_system"
> checking which qtsingleapplication to use... shipped
> checking for libtorrent... no
> configure: error: Package requirements (libtorrent-rasterbar >= 1.0.6) were not                                                                                         met:
> 
> Requested 'libtorrent-rasterbar >= 1.0.6' but version of libtorrent-rasterbar is                                                                                        
> 
> Consider adjusting the PKG_CONFIG_PATH environment variable if you
> installed software in a non-standard prefix.
> 
> Alternatively, you may set the environment variables libtorrent_CFLAGS
> and libtorrent_LIBS to avoid the need to call pkg-config.
> See the pkg-config man page for more details.
> make: *** No targets specified and no makefile found.  Stop.
> 

 
> ● Error: The build command produced an exit code greater than 0 - Check the log                                                                                        s

""


[qbittorrent.log.txt](https://github.com/userdocs/qbittorrent-nox-static/files/6950080/qbittorrent.log.txt)
[libtorrent.log.txt](https://github.com/userdocs/qbittorrent-nox-static/files/6950081/libtorrent.log.txt)

**Reply by userdocs** [reply]
I had a quick look at this. I can fix the initial error but i get linking issue with qbit that i am not sure i want to try ans resolve. I don't think i really built before version 4.2.* tbh.

So i am not sure i can be fixed. Does it have to be that version?

**Reply by Sparklingx** [reply]
@userdocs  Qbittorrent version  can be any version as long as the libtorrent version is 1.1.14.   And you are right the build of 4.1.9.1 is by "jerry048" using your script I am assuming but not sure. And also thanks a lot for having a look into it. ❤

**Reply by userdocs** [reply, marked as answer]
Ok, so i just realised why it stopped working when fixing something else. The version of boost defaults to the most recent but this breaks older build combos. I have fixed it by allowing you to specify a boost version. This command should work. It did in my testing.

```
~/qbittorrent-nox-static.sh all -qt release-4.1.9.1 -lt libtorrent-1_1_14 -bv 1.76.0
```

**Reply by Sparklingx** [reply]
> Ok, so i just realised why it stopped working when fixing something else. The version of boost defaults to the most recent but this breaks older build combos. I have fixed it by allowing you to specify a boost version. This command should work. It did in my testing.
> 
> ```
> ~/qbittorrent-nox-static.sh all -qt release-4.1.9.1 -lt libtorrent-1_1_14 -bv 1.76.0
> ```


this works now. thanks
